### PR TITLE
msg/async misc cleanup 

### PR DESCRIPTION
**Pull request by majianpeng**

### CODE REVIEW

File: src/msg/async/AsyncConnection.cc [hunk 1]
```diff
@@ -2212,13 +2210,10 @@ ssize_t AsyncConnection::write_message(Message *m, bufferlist& bl, bool more)
   ssize_t rc = _try_send(more);
   if (rc < 0) {
     ldout(async_msgr->cct, 1) << __func__ << " error sending " << m << ", "
-                              << cpp_strerror(rc) << dendl;
-  } else if (rc == 0) {
-    logger->inc(l_msgr_send_bytes, total_send_size - original_bl_len);
-    ldout(async_msgr->cct, 10) << __func__ << " sending " << m << " done." << dendl;
+      << cpp_strerror(rc) << dendl;
```
Comment (yuyuyu101) [inline src/msg/async/AsyncConnection.cc]: aligned

**Comment by yuyuyu101** [comment]
@majianpeng I don't find the bug fix, it's all cleanup, right?

**Comment by majianpeng** [comment]
@yuyuyu101 . update by your suggestion. Also update PR comments. Thanks!

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-10-25_20:20:08-rados-wip-sage2-testing-2017-10-25-1347-distro-basic-smithi/

**Pull request merged**